src/OpenLoco/src/Graphics/SoftwareDrawingEngine.cpp

```diff
@@ -315,37 +315,41 @@ namespace OpenLoco::Gfx
             SDL_UnlockSurface(_screenSurface);
         }
 
-        // Convert colours via palette mapping onto the RGBA surface.
-        if (SDL_BlitSurface(_screenSurface, nullptr, _screenRGBASurface, nullptr))
+        if (Config::get().scaleFactor > 1.0f)
```

Comment:
Should there be a note here about avoiding this when no scaling for performance reasons

Reply:
I've added a comment.